### PR TITLE
Quick-Facts 'en'/'es' 5/20/2019 updated

### PR DESCRIPTION
**Pull request by lh1008**
Hey @erciccione,

Added the new updated 5/20/2019 Quick-Facts Sheet version for 'en' and 'es'. This needs a fast review/merge so we can work out the translation as fast as we can, due to updates occur every ~3-4 months so we have time to share throughout the community.

- Added md, pdf files, and sla file for English only.

>:)

Pinging @BlackLotus64 for Spanish review and if he can help us out with english would be great.

### CODE REVIEW

**Comment by erciccione** [comment]
Waiting for @BlackLotus64's review.

If you want this to be merged quickly i suggest to open 2 different Pull Request, one with the surce text in English and another one with the Spanish translations, so that we can merge the english text straight away if it's urgent

**Comment by BlackLotus64** [comment]
Can do both today (if they're not too much of an update), will be checking it when I get back home.

**Comment by lh1008** [comment]
Ok for the next update will do different PR for easier merging. Thanks.

@BlackLotus64, there aren't many changes, let me list them here so you can go straight to the updates:

- Key Differentiating Factors -> Monero Research Lab
- Key Differentiating Factors -> Capitalize Titles (only English, in Spanish, capitalize every word is not used)
- Technical Fundamentals
- Features in Development

File: monero-outreach-docs/en/quickfacts/Quickfacts.md [hunk 1]
```diff
@@ -122,7 +120,7 @@ Although Monero is already available and being used across the globe, the commun
 + **_[Mastering Monero](https://masteringmonero.com/)_**
 + **_[Satis Group Report](https://research.bloomberg.com/pub/res/d37g1Q1hEhBkiRCu_ruMdMsbc0A)_**
 
-**Monero Quick Facts - Revised 2/1/2019** 
+**Monero Quick Facts - Revised 5/2/2019**
```
Comment (BlackLotus64) [inline monero-outreach-docs/en/quickfacts/Quickfacts.md]: Just to confirm, should't it be 5/20 as in the technical fundamentals?

File: monero-outreach-docs/translations/es/quickfacts/QuickFacts_es.md [hunk 2]
```diff
@@ -67,7 +67,7 @@ Quiere decir que la palabra Monero se puede definir como: “verbo que describe
...
 
-**El Laboratorio de Investigación de Monero:** Monero no sólo está comprometido en hacer una moneda fungible, sino también en la continua investigación en el campo de la privacidad financiera que involucra a las criptomonedas en general. Con ese fin, investigadores académicos en campos de matemáticas, física, seguridad y computación en blockchain han publicado **[libros blancos](https://lab.getmonero.org/)** y tienen muchas otras **[metas de investigación](https://getmonero.org/es/design-goals/)** en las que están trabajando.
+**El Laboratorio de Investigación de Monero:** Monero no sólo está comprometido en hacer una moneda fungible, sino también en la continua investigación en el campo de la privacidad financiera que involucra a las criptomonedas en general. El [Laboratorio de Investigación de Monero](https://web.getmonero.org/es/resources/research-lab/) (MRL-siglas en inglés) es un grupo de investigadores académicos en campos de las matemáticas, la física, seguridad y la computación de la tecnología blockchain quienes investigan soluciones para Monero y publican documentos académicos con sus hallazgos.
```
Comment (BlackLotus64) [inline monero-outreach-docs/translations/es/quickfacts/QuickFacts_es.md]: A little suggest here:

_El Laboratorio de Investigación de Monero (MRL por sus siglas en inglés) es un grupo de investigadores académicos en los campos de matemáticas, física, seguridad y computación de la tecnología blockchain, quienes investigan soluciones para Monero y publican documentos académicos con sus hallazgos._

File: monero-outreach-docs/translations/es/quickfacts/QuickFacts_es.md [hunk 3]
```diff
@@ -97,11 +97,11 @@ _Con una emisión definitiva (tail emission) de 0.6 XMR/bloque, para el 2040 hab
...
+**RandomX:** Es un nuevo tipo de algoritmo de prueba-de-trabajo (PoW) diseñado para prevenir la centralización de la minería por los ASICs y estimular la minería igualitaria en CPU para asegurar la red. RandomX está en el proceso de auditoría y se espera que entre a la red en la próxima actualización.
 
-**Monederos físicos (hardware):** Los famosos monederos Ledger y Trezor están trabajando en agregar a Monero (Ledger es soportado desde los monederos CLI 0.12.2 y GUI 0.12.3). La comunidad de Monero está financiando a un equipo de trabajo que está desarrollando **[un monedero físico](http://kastelo.org/)** construido sobre las características distintivas de Monero. Ambos monederos Ledger y Trezor fueron anticipados de ser operacionales a finales del 2018.
+**Limpieza de la blockchain (pruning):** Monero, para facilitar la escabilidad, recientemente agrego la característica de la limpieza de la blockchain a su software. Ésta característica le permite a los usuarios "limpiar", de manera óptima, dos terceras partes de la información de la blockchain mientras que se sigue contribuyendo a la red.
```
Comment (BlackLotus64) [inline monero-outreach-docs/translations/es/quickfacts/QuickFacts_es.md]: + **agregó**
+ **Esta**, no accent.

File: monero-outreach-docs/en/quickfacts/Quickfacts.md (hunk 1 above)
Comment (BlackLotus64) [reply monero-outreach-docs/en/quickfacts/Quickfacts.md]: Yes it should, all the other dates are 5/20.

Comment (lh1008) [reply monero-outreach-docs/en/quickfacts/Quickfacts.md]: fixed

File: monero-outreach-docs/translations/es/quickfacts/QuickFacts_es.md (hunk 2 above)
Comment (lh1008) [reply monero-outreach-docs/translations/es/quickfacts/QuickFacts_es.md]: fixed

File: monero-outreach-docs/translations/es/quickfacts/QuickFacts_es.md (hunk 3 above)
Comment (lh1008) [reply monero-outreach-docs/translations/es/quickfacts/QuickFacts_es.md]: fixed

File: monero-outreach-docs/en/quickfacts/Quickfacts.md (hunk 1 above)
Comment (lh1008) [reply monero-outreach-docs/en/quickfacts/Quickfacts.md]: The pdf and .sla file didn't have this error. It was the .md that had it. Thank you for the notice :).

**Comment by lh1008** [comment]
The Spanish pdf was fixed with the suggested changes, the English version didn't have the date error, I think I noticed it in the sla file but not in the md file. 

Thank you @BlackLotus64 it should be ready for review.

**Pull request merged**